app/interactor/snippet_exec_interactor.go

```diff
@@ -23,14 +23,22 @@ func NewSnippetExecInteractor(
 	}
 }
 
+type execNoFileError struct {
+	msg string
+}
+
+func (e *execNoFileError) Error() string {
+	return fmt.Sprintf("error: %s", e.msg)
+}
```

Comment:
Please fix as following:

```suggestion
func (e *execNoFileError) Error() string {
	return "no files in gist entry"
}
```

Not only this error, but also all of errors as well.

Reply:
かしこまりました。修正します。
局所的に使用するエラーなので、エラー文を変更できるようにする必要が無いということですね。